File: src/main/java/ru/abe/slaves/potrebot/web/model/VkEvent.java
package ru.abe.slaves.potrebot.web.model;

import com.fasterxml.jackson.annotation.JsonInclude;
import com.fasterxml.jackson.annotation.JsonProperty;
import lombok.Data;

@Data
public class VkEvent {
    private String type;
    private VkEventContent object;
    @JsonProperty("group_id")
    private long groupId;
    @JsonProperty("event_id")
    private String eventId;
}

File: src/main/java/ru/abe/slaves/potrebot/web/model/VkEvent.kt
package ru.abe.slaves.potrebot.web.model

import com.google.gson.annotations.SerializedName

data class VkEvent(
    val type: String?,
    @SerializedName("object") val content: VkEventContent,
    @SerializedName("group_id") val groupId: Long = 0,
    @SerializedName("event_id") val eventId: String
)